pola/templates/partiales/friends.html

```diff
@@ -1,7 +1,7 @@
         <h2>Wspieramy polski biznes! Oto nasi przyjaciele:</h2>
 
         <div class="section">
-            <p><strong>1 RADZIEMSKA</strong></p>
+            <p><strong><li><a href="https://www.radziemska.pl/">1. Radziemska</a></li></strong></p>
```

Comment:
Dlaczego potrzebujemy <li> znacznika w tym miejscu?
Ja już pracuje nad podobną zmianą, ale nie mogłem znaleźć chwili, aby skończyć. Dzisiaj na to spojrzę.